app/main.py

```diff
@@ -1,6 +1,23 @@
+import requests
+
+
 def get_weather() -> None:
-    # write your code here
-    pass
+    """Get weather data from an external API and print the result."""
+    api_key = "b54a45c623e74410aed183111231004"
```

Comment:
don't hardcode api_key, it's sensitive data
u should add it to the environment variables
The user should not have access to your program if he does not have an API key
![image](https://user-images.githubusercontent.com/86322722/231115431-52c58b11-e05a-4d99-9990-3f4823dcbb48.png)